### DISCUSSION TITLE
Is there a utils extension that allows you to modify the JSON file?

### DISCUSSION THREAD

**Discussion by akuznetsov-congree** [wixtoolset/issues#7711] [Questions]
Wix has a Util extension that allows you to work with XML files. But a modern application contains many json files. Is there any extension for Wix 4 that is responsible for changing the JSON file? Maybe it makes sense to add an extension to Utils for working with json?

**Comment by nirbar** [comment]
My WiX extension supports it- https://www.nuget.org/packages/PanelSwWixExtension4 (WiX v4) or https://www.nuget.org/packages/PanelSwWixExtension (WiX v3):
```
      <Component Directory="INSTALLFOLDER">
        <File Id="Json_Object.json" Source="$(var.ProjectDir)Json-Object.json">
          <PanelSW:JsonJPath JPath="$.Manufacturers[\[]?(@.Name == 'MyName')[\]].Customer1" Value="TrUE" Formatting="boolean" />
        </File>
      </Component>
```

**Reply by EndOfCode** [reply]
Hi @nirbar,
which namespace do I have to set for PanelSW?

**Comment by nirbar** [comment]
http://schemas.panel-sw.co.il/wix/WixExtension

**Reply by EndOfCode** [reply]
Thank you. Works well.
